cernopendata/modules/fixtures/data/articles/cms-guide-for-research/research.md

```diff
@@ -127,35 +128,34 @@ If you are interested in conducting a more sophisticated analysis using CMS Open
 
 > "I want to find out how to use the trigger and trigger prescale information in the dataset I am interested in."
 
-- Go to CMS Trigger Information (currently for 2011 only).
-- This information is still very basic, and will be improved.
+- Go to [the Guide to CMS trigger system](http://opendata-dev.web.cern.ch/articles/guide-to-the-cms-trigger-system).
 
 <center>–––</center>
 
 > "I want to find out how to access the luminosity information for the dataset I am interested in and how to select "good data" only."
 
-- Check the [luminosity record](/record/1051), and
-- check the [JSON description](/record/1000).
+- Check the [CMS luminosity information](http://opendata-dev.web.cern.ch/search?page=1&size=20&q=luminosity&type=Supplementaries&subtype=Luminosity), and
+- check the [list of validated runs](http://opendata-dev.web.cern.ch/search?page=1&size=20&q=%22CMS%20list%20of%20validated%20runs%22).
 
 <center>–––</center>
 
 > "I want to find out whether I need condition data base information, and if so, how to access it."
 
 - Condition data are needed only on sophisticated examples using e.g. jet energy corrections (many of the simpler analysis/validation examples documented here do not use this).
 - Using condition data significantly slows down data access, so use them only if really needed. If so:
-    - go to CMS Condition Data for basic information (in contrast to what is stated there, they are NOT needed to run CERNVM in general, only to perform sophisticated tasks).
+    - see the instructions under to Option B in [Getting started with CMS 2011 data](CMS Condition Data for basic information (in contrast to what is stated there, they are NOT needed to run CERNVM in general, only to perform sophisticated tasks).
 - This information is still very basic and will be improved.
 
 <center>–––</center>
 
 > "I want to find the luminosity of my dataset, possibly constrained by using specific triggers."
 
-- Please check [CMS luminosity information](/record/1051),
+- Please check [CMS luminosity information](http://opendata-dev.web.cern.ch/search?page=1&size=20&q=luminosity&type=Supplementaries&subtype=Luminosity),
 - decide on the triggers you want to use,
 - find out the runs/lumi sections in which these triggers were active and whether they were prescaled,
 - overlap with the available CMS Open Data samples (run range) and with the JSON data-quality selection.
     - If not prescaled, sum up the luminosity for the surviving runs/lumi sections.
-    - If prescaled, life is more complicated…
+    - If prescaled, life is more complicated, find the prescales as in [the trigger example code](FIXME)
```

Comment:
`- If prescaled, life is more complicated. Find the prescales as in [the trigger example code](FIXME)`